### DISCUSSION TITLE
Options indicators

### DISCUSSION THREAD

**Discussion by lolrenx** [OpenBB-finance/OpenBB#175] [Ideas]
Hey guys, 

what do you think about adding options related info such as open interest, contract volumes, max pain strike etc …?

I am fairly new to coding, but happy to get the ball rolling on that

**Comment by DidierRLopes** [comment]
Hey @lolrenx this is a really good idea! Options is a world, and I'm not that familiar into the area. I know that some of the APIs that are currently being used, can get options info, but since I'm not too experienced in the area, I wasn't sure about the best way to present the information to the user! Feel free to say a list of features that you think would be good to have and we can discuss it in here, and then you can work on a new section for options based on the feedback discussed here!

**Reply by lolrenx** [reply]
sounds great, I'll get started on that. 
ideal starters are open interest and max pain, both can help project sp movements due to market makers gamma hedging :)

**Reply by DidierRLopes** [reply]
Feel free to create a PR against our ROADMAP: https://github.com/DidierRLopes/GamestonkTerminal/blob/main/ROADMAP.md

With your proposed features, and then assign it to you, by adding (@lolrenx ) at the end. This way people will know that there's someone working on such features :)
